packages/cli-kit/src/public/node/environment/fqdn.ts

```diff
@@ -50,32 +52,14 @@ export async function identity(): Promise<string> {
 }
 
 /**
- * It returns the Shopify service we should interact with.
- * Note the same fqdn is sued for the Admin and the Storefront Renderer APIs.
- * @returns Fully-qualified domain of the Shopify service we should interact with.
+ * Normalize the store name to be used in the CLI.
+ * It will add the .myshopify.com domain if it's not present.
+ * It will add the spin domain if it's not present and we're in a Spin environment.
+ *
+ * @param store - Store name.
+ * @returns Normalized store name.
  */
-export async function shopify(options: {storeFqdn?: string} = {}): Promise<string> {
-  const environment = serviceEnvironment()
-  switch (environment) {
-    case 'local':
-      return 'shopify.myshopify.io'
-    case 'spin':
-      return `identity.${await spinFqdn()}`
-    default:
-      if (options.storeFqdn) {
-        return options.storeFqdn
-      } else {
-        throw NotProvidedStoreFQDNError
-      }
-  }
-}
-
-/**
- * Given a store, returns a valid store fqdn removing protocol and adding the proper domain in case is missing
- * @param store - Original store name provided by the user
- * @returns a valid store fqdn
- */
-export async function normalizeStoreName(store: string) {
+export async function normalizeStoreName(store: string): Promise<string> {
```

Comment:
i'd say is not exactly `storeNameFromURI` but more `storeFQDNFromName` 🤔  
or even `normalizeStoreFqdn`, because this method will always return a valid fqdn